src/tsickle.ts

```diff
@@ -1216,6 +1216,15 @@ class Annotator extends ClosureRewriter {
       if (sym.flags & ts.SymbolFlags.Alias) {
         sym = this.typeChecker.getAliasedSymbol(sym);
       }
+      // tsickle does not emit exports for ambient namespace declarations:
+      //    "export declare namespace {...}"
```

Comment:
Sorry, this keeps coming up again, but I need to get my terminology straight - you are saying this namespace is ambient because it has a 'declare'? I previously thought 'ambient' means not in a module, which cannot be true here because you have an 'export' keyword.

Also in the clutzed.d.ts there is no export

Reply:
declare means ambient. export means it's in a proper module as oppose to a namespace. I'd call a pure namespace that's not nested in a module "global".

This fix is actually covered by a different pre existing test that was tickled by the other change, the .d.ts covers a different case.